dxa-framework/dxa-common-impl/src/main/java/com/sdl/webapp/common/impl/taglib/xpm/XpmButtonTag.java

```diff
@@ -25,6 +26,19 @@ public void setRegion(RegionModel region) {
         this.region = region;
     }
 
+    /**
+     * <p>Setter for the field <code>cssClass</code>.</p>
+     *
+     * @param cssClass a {@link string} object.
+     */
+    public void setCssClass(String cssClass) {
```

Comment:
I believe this logic should not be in setter but in a view. Also it's not covered with unit test